src/group_theory/group_action/symmetric.lean

```diff
@@ -0,0 +1,112 @@
+/-
+Copyright (c) 2021 Jakob von Raumer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakob von Raumer, Eric Wieser
+-/
+import algebra.opposites
+import group_theory.group_action.defs
+import group_theory.group_action.prod
+import algebra.direct_sum.basic
+
+/-!
+# Symmetric group actions
+
+This file defines a class of group actions which are invariant unter taking the `opposite` of
+the operating type/(semi)group/monoid.
+
+The main definition is the one of the typeclass `is_symmetric_smul` which states that for a
+pair of types `R` and `M`, the actions of `R` and its opposite `Rᵐᵒᵖ` on `M` coincide.
+
+We then provide the alternate version of its constructor, `unop_smul_eq_smul` about general
+instance of `Rᵐᵒᵖ` and several instances for closure properties.
+
+## Tags
+
+group action
+-/
```

Comment:
Note that #10543 ended up calling this `is_central_scalar` instead, and distributed  some of the defs in this file to other places. Some of the lemmas in this file did not make it into that PR (`units.is_symmetric_smul` can't be stated without other instances that are new in this PR, for instance), so it's not quite as easy as deleting this file. Hopefully simply replacing `is_symmetric_smul` with `is_central_scalar` will do most of the work, and then lean will tell you which definitions are duplicates.